src/components/BotBuilder/ContactBot.js

```diff
@@ -18,6 +18,19 @@ class ContactBot extends React.Component {
 	}
 
 	render() {
+		if(!cookies.get('loggedIn'))
+        {
+            return (
+                <div>
+                    <StaticAppBar {...this.props} />
+                    <div>
+                        <p style={styles.loggedInError}>
+                            Please login create the Contact Bot.
```

Comment:
Make it "Please login to create the Contact Bot" :sweat_smile:

Reply:
Thanks. Made the change.